monitoring/monitoring-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="monitoring-overview"]
+= Monitoring overview
+include::modules/common-attributes.adoc[]
+:context: monitoring-overview
+
+toc::[]
+
+[id="about-openshift-container-platform-monitoring"]
+== About {product-title} Container Platform monitoring
+
+{product-title} includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects]. {product-title} delivers monitoring best practices out of the box. A set of alerts are included by default that immediately notify cluster administrators about issues with a cluster. Default dashboards in the {product-title} web console include visual representations of cluster metrics to help you to quickly understand the state of your cluster.
```

Comment:
The sentence, "A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations" could come after "A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations." I am suggesting that because the page on configuring the monitoring stack relates to the core OpenShift monitoring components as well as monitoring for user-defined projects.

Reply:
You could remove the bold weighting from "monitoring for core platform components". With the changes in this PR, "monitoring for user-defined projects" is no longer bold, because it is now a link. My view is that having only one of these in bold does not add value when scanning the section.

Reply:
You could change "enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects]" to "xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects]", so that "enable" is part of the link.

Reply:
@pneedle-rh

Reply:
Ah yes, my apologies. My suggestion is that you could restructure those sentences as follows:

> {product-title}` includes a pre-configured, pre-installed, and self-updating monitoring stack that provides *monitoring for core platform components*. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects. A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations.